web/modules/custom/usagov_ssg_postprocessing/usagov_ssg_postprocessing.module

```diff
@@ -44,10 +46,12 @@ function usagov_ssg_postprocessing_modify_canonical_link(array &$attachments) {
 }
 
 /**
- * Implements hook_page_attachments_alter.
- * We just want to add / to the alternate links on the home pages.
+ * Implements hook_page_attachments_alter
+ *
+ * @param array<mixed> &$attachments
```

Comment:
Looking at the code, `$attachments` is expected to be a string-indexed array: `array<string, mixed>`